build/custom.mk

```diff
@@ -19,4 +25,4 @@ ifndef MM_RUDDER_WRITE_KEY
     MM_RUDDER_WRITE_KEY = 1d5bMvdrfWClLxgK1FvV3s4U1tg
```

Comment:
It seems there's a remnant reference to `MM_RUDDER_WRITE_KEY` here

Reply:
nice catch!